controller/engine_controller.go

```diff
@@ -1402,6 +1396,7 @@ func (ec *EngineController) startRebuilding(e *longhorn.Engine, replica, addr st
 	if err != nil {
 		return err
 	}
+	defer engineClientProxy.Done()
```

Comment:
This should be moved to the below go routine to ensure replica rebuilding successfully.

Reply:
Fixed.